apps/src/p5lab/spritelab/CoreLibrary.js

```diff
@@ -148,7 +148,8 @@ export default class CoreLibrary {
         return;
       }
 
-      const value = this.getVariableValue(name);
+      // Use variable value or empty string (if undefined).
+      const value = this.getVariableValue(name, '');
```

Comment:
This is the original use case for variable bubbles. We want to us an empty string for undefined values.